src/NexusMods.App.UI/Pages/LibraryPage/LibraryViewModel.cs

```diff
@@ -155,57 +139,28 @@ public LibraryViewModel(
                 }
 
                 vm.InstallCommandDisposables.Clear();
-            });
+            }).AddTo(disposables);
 
-            ActivationSubject
+            Adapter.ModelActivationSubject
                 .Subscribe(this, static (tuple, vm) =>
                 {
                     var (model, isActivated) = tuple;
 
                     if (isActivated)
                     {
-                        model.Ticker = vm._ticker;
                         var disposable = model.InstallCommand.Subscribe(vm, static (libraryItemId, vm) => vm.InstallLibraryItemSubject.OnNext(libraryItemId));
                         var didAdd = vm.InstallCommandDisposables.TryAdd(model, disposable);
                         Debug.Assert(didAdd, "subscription for the model shouldn't exist yet");
-
-                        model.Activate();
                     }
                     else
                     {
-                        model.Ticker = null;
                         var didRemove = vm.InstallCommandDisposables.Remove(model, out var disposable);
                         Debug.Assert(didRemove, "subscription for the model should exist");
                         disposable?.Dispose();
-
-                        model.Deactivate();
                     }
                 })
                 .AddTo(disposables);
```

Comment:
I don't particularly like this dictionary of disposables that we have to maintain for subscribing on child entries commands. It seems an overengineered solution.

I do agree that it is preferrable to deal with the commands from the parent VM. 
Maybe we can come up with a better approach